docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValuesMultiColumn.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class ReplaceMissingValuesMultiColumn
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+            var samples = new List<DataPoint>()
+            {
+                new DataPoint(){ Label = 3, Features1 = new float[3] {1, 1, 0}, Features2 = new float[2] {1, 1} },
+                new DataPoint(){ Label = 32, Features1 = new float[3] {0, float.NaN, 1}, Features2 = new float[2] {0, 1} },
+                new DataPoint(){ Label = 5, Features1 = new float[3] {-1, 2, -3}, Features2 = new float[2] {-1, float.NaN} },
+                new DataPoint(){ Label = 9, Features1 = new float[3] {-1, 6, -3}, Features2 = new float[2] {0, float.PositiveInfinity} },
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
+            var meanPipeline = mlContext.Transforms.ReplaceMissingValues(new[] {
+                new InputOutputColumnPair("MissingReplaced1", "Features1"),
+                new InputOutputColumnPair("MissingReplaced2", "Features2")
+            },
+            MissingValueReplacingEstimator.ReplacementMode.Mean);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var meanTransformer = meanPipeline.Fit(data);
+            var meanTransformedData = meanTransformer.Transform(data);
+
+            // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
+            var meanRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(meanTransformedData, reuseRowObject: false);
+
+            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
+            var defaultPipeline = mlContext.Transforms.ReplaceMissingValues(new[] {
+                new InputOutputColumnPair("MissingReplaced1", "Features1"),
+                new InputOutputColumnPair("MissingReplaced2", "Features2")
```

Comment:
optional: we don't have an in-place transformation multicolumn sample. can we transform the data in-place and not have MissingReplaced1/MissingReplaced2?